lib/elixir/lib/calendar/time.ex

```diff
@@ -592,11 +592,13 @@ defmodule Time do
     }
   end
 
-  defp new_duration!(%Duration{} = duration) do
+  @doc false
+  def new_duration!(%Duration{} = duration) do
```

Comment:
We should use `__` so they are not accidentally imported. Maybe `__duration__!`?

Reply:
Done in [40e8d10](https://github.com/elixir-lang/elixir/pull/13489/commits/40e8d106594bc5bd75dd56616a9fa322aae4b4e6)